lib/sendgrid/helpers/inbound/README.md

```diff
@@ -17,7 +17,7 @@
 # Installation
 
 In addition to the installation instructions in
-[the main readme](https://github.com/sendgrid/sendgrid-ruby/tree/master/#installation),
+[the main readme](#installation),
```

Comment:
No links are working.

Reply:
[These links](https://github.com/sendgrid/sendgrid-ruby/blob/09752f9d90c68ec89d7fb35ed7b0f4757026d39f/lib/sendgrid/helpers/inbound/README.md) are all working for me.

Reply:
Then the link makes no sense. It's referencing the anchor from 3 lines up?